com.unity.formats.alembic/Runtime/Scripts/Importer/AbcAPI.cs

```diff
@@ -148,6 +150,8 @@ struct aiMeshSummary
         public Bool hasTangents { get; set; }
         public Bool hasUV0 { get; set; }
         public Bool hasUV1 { get; set; }
+
+        public int hasAttributes { get; set; }
```

Comment:
[ABC-486](https://jira.unity3d.com/browse/ABC-486) : This will contain the number of custom attributes. We fill this struct just before allocating the memory to the attributes in C#, and after reading the data from the file in the C++ side.

Reply:
```suggestion
        public int attributesCount { get; set; }
```